diff -u -r a/dsdt.dsl b/dsdt.dsl
--- a/dsdt.dsl	2021-08-11 18:51:41.132545981 -0400
+++ b/dsdt.dsl	2021-08-11 18:54:51.855873888 -0400
@@ -18,7 +18,7 @@
  *     Compiler ID      "INTL"
  *     Compiler Version 0x20190509 (538510601)
  */
-DefinitionBlock ("", "DSDT", 2, "ALASKA", "A M I ", 0x01072009)
+DefinitionBlock ("", "DSDT", 2, "ALASKA", "A M I ", 0x01072010)
 {
     External (_SB_.ALS_, DeviceObj)
     External (_SB_.ALS_.LUXL, UnknownObj)
@@ -3117,7 +3117,7 @@
         Zero, 
         Zero
     })
-    Name (XS3, Package (0x04)
+    Name (_S3, Package (0x04)
     {
         0x03, 
         Zero, 
@@ -11886,11 +11886,6 @@
 
     Scope (_SB.PCI0.GP17.XHC0)
     {
-        Method (_S0W, 0, NotSerialized)  // _S0W: S0 Device Wake State
-        {
-            Return (0x03)
-        }
-
         Device (RHUB)
         {
             Name (_ADR, Zero)  // _ADR: Address
@@ -12102,11 +12097,6 @@
 
     Scope (_SB.PCI0.GP17.XHC1)
     {
-        Method (_S0W, 0, NotSerialized)  // _S0W: S0 Device Wake State
-        {
-            Return (0x03)
-        }
-
         Device (RHUB)
         {
             Name (_ADR, Zero)  // _ADR: Address
